data/www-widgetapi.js

```diff
@@ -0,0 +1,159 @@
+(function(){var g,k=this;function l(a){a=a.split(".");for(var b=k,c;c=a.shift();)if(null!=b[c])b=b[c];else return null;return b}
```

Comment:
This is youtube's iframe api lib. I decided to check it in order to simplify and fix #258 . We also have once less http request and we are a bit more secure.